circle.yml

```diff
@@ -15,8 +15,8 @@ dependencies:
     - docker pull plotly/imageserver:latest
   post:
     - npm run cibuild
-    - docker run -d --name myimageserver -v $PWD:/var/www/streambed/image_server/plotly.js -p 9010:9010 plotly/imageserver:latest; sleep 20
-
+    - docker run -d --name myimageserver -v $PWD:/var/www/streambed/image_server/plotly.js -p 9010:9010 plotly/imageserver:latest
+    - wget --server-response --spider --tries=8 --retry-connrefused http://localhost:9010/ping
```

Comment:
An even better script would restart `nw1` if the image server doesn't return a `pong` after 8 tries.

Looks like the latest container is reliable enough that we don't need this now. But, maybe later.